python/dgl/nn/pytorch/conv/gatedgcnconv.py

```diff
@@ -0,0 +1,173 @@
+"""Torch Module for GatedGCN layer"""
+# pylint: disable= no-member, arguments-differ, invalid-name, cell-var-from-loop
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+from .... import function as fn
+
+
+class GatedGCNConv(nn.Module):
+    r"""Gated graph convolutional layer from `Benchmarking Graph Neural Networks
+    <https://arxiv.org/abs/2003.00982>`__
+
+    .. math::
+        e_{ij}^{l+1}=D^l h_{i}^{l}+E^l h_{j}^{l}+C^l e_{ij}^{l}
+
+        norm_{ij}=\Sigma_{j\in N_{i}} \sigma\left(e_{ij}^{l+1}\right)+\varepsilon
```

Comment:
In the original paper, it should be

```
norm_{ij}=\Sigma_{j\in N_{i}} \sigma\left(e_{ij}^{l+1}\right)+\varepsilon+e_{ij}^{l}

\hat{e}_{ij}^{l+1}=\sigma(e_{ij}^{l+1} + e_{ij}^{l}) / norm_{ij}
```
right?

Reply:
Also you interchange the definition of \hat{e} and e, which can be confusing. It's better to be consistent with the paper notations.